src/scripts/wordle_guesser.js

```diff
@@ -21,9 +22,11 @@ class WordleGame {
       if (guess[i] === this.secret[i]) {
         this.answer[i] = char;
         this.possibleLetters[i] = [char];
+        this.confirmedLetters.push(char);
```

Comment:
just a thought, might be worth renaming `possibleLetters` and `confirmedLetters` to something like `possibleLettersAtPosition` and `confirmedLettersInWord`? longer but clearer

Reply:
Renamed!